libs/ui/src/components/NeoModal/NeoModalBody.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="modal-width">
+    <header
+      class="px-6 is-flex is-justify-content-space-between border-bottom is-align-items-center"
+      :class="{
+        'py-5': !loading,
+        'py-4': loading,
+      }">
+      <NeoSkeleton
+        v-if="loading"
+        rounded
+        width="150"
+        height="35"
+        no-margin
+        border-radius="4rem"
+        variant="k-grey-light" />
+
+      <transition name="fade">
+        <span
+          v-if="!loading"
+          class="modal-card-title is-size-6 has-text-weight-bold">
+          {{ title }}
+        </span>
+      </transition>
+
+      <NeoButton
+        variant="text"
+        no-shadow
+        icon="xmark"
+        size="medium"
+        @click="onClose" />
+    </header>
+
+    <div
+      class="is-relative"
+      :class="[
+        {
+          'limit-height__scrollabe': scrollable,
+          'limit-height__loading': loading,
+        },
+        `px-${paddingX}`,
+        `pt-${paddingTop}`,
+        `pb-${paddingBottom}`,
+      ]">
+      <div v-if="loading" class="skeleton-container">
+        <NeoSkeleton
+          class="skeleton-backdrop"
+          rounded
+          border-radius="20px"
+          no-margin
+          full-size
+          variant="k-grey-light" />
+
+        <div class="skeleton-content is-flex">
+          <NeoIcon
+            icon="spinner-third"
+            class="spinner has-text-k-grey mr-6"
+            size="large"
+            spin></NeoIcon>
+
+          <div>
+            <p class="is-capitalized has-text-weight-bold is-size-6">
+              {{ $t('general.doingSomeMagic') }}
+            </p>
+            <p class="is-capitalized is-size-6 has-text-k-grey">
+              {{ $t('general.pleaseWait') }}
+              <span class="dots" />
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div
+        ref="slot"
+        class="slot"
+        :class="{
+          slot__loading: loading,
+        }">
+        <slot />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import NeoSkeleton from '../NeoSkeleton/NeoSkeleton.vue'
+import NeoIcon from '../NeoIcon/NeoIcon.vue'
+import NeoButton from '../NeoButton/NeoButton.vue'
+
+const emits = defineEmits(['close'])
+withDefaults(
+  defineProps<{
+    title: string
+    loading?: boolean
+    modalWidth?: string
+    modalMaxHeight?: string
+    paddingTop?: string
+    paddingBottom?: string
+    paddingX?: string
```

Comment:
use content-class instead?

Reply:
content-class  is for neomodal not for this section 
![CleanShot 2023-11-27 at 18 28 46@2x](https://github.com/kodadot/nft-gallery/assets/44554284/d5982ac2-b3a7-40e1-bfdf-c1609c0531a1)

which is the bottom section of the modal 
![CleanShot 2023-11-27 at 18 29 16@2x](https://github.com/kodadot/nft-gallery/assets/44554284/9f7fa523-62ef-4a18-91ff-7e54a3f332f8)

Reply:
This clearly doesn't look right, I find this to specific for a single case.

Maybe we should instead style the content, like how it's done in [Buefy](https://buefy.org/documentation/modal/#component) (modal-card)

Reply:
fair enough